callautomation_appointmentbooking/resources/config.conf

```diff
@@ -0,0 +1,25 @@
+[APPOINTMENT_BOOKING_CONFIG]
```

Comment:
This configuration file format is pretty simple and does not allow for values between quotes. This is misleading as ACS connection strings are quite "weird", and the IDE syntax highlighting may make us believe to add quotes, resulting in an invalid connection string.

IMHO the format of the config file should be changed:
```
ACS_CONNECTION_STRING=endpoint=https://....
```
is more confusing than
```
ACS_CONNECTION_STRING="endpoint=https://..."
```